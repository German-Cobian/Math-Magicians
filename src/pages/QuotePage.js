import React from 'react';
import './Pages.css';

const Quote = () => (
  <div className="quotepageContainer">
    <p className="quoteText">Mathematics is the music of reason.</p>
    <p className="quoteText"><strong>â€” James Joseph Sylvester</strong></p>
  </div>
);

export default Quote;
