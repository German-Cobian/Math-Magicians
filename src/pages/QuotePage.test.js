import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import QuotePage from './QuotePage';

describe('<QuotePage />', () => {
  it('it should match snapshot', () => {
    const tree = renderer.create(<QuotePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
