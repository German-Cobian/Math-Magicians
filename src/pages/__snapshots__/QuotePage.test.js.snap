// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuotePage /> it should match snapshot 1`] = `
<div
  className="quotepageContainer"
>
  <p
    className="quoteText"
  >
    Mathematics is the music of reason.
  </p>
  <p
    className="quoteText"
  >
    <strong>
      â€” James Joseph Sylvester
    </strong>
  </p>
</div>
`;
